public/app/features/trails/MetricSelect/api.ts

```diff
@@ -1,12 +1,15 @@
 import { AdHocVariableFilter, RawTimeRange, Scope } from '@grafana/data';
 import { getPrometheusTime } from '@grafana/prometheus/src/language_utils';
+import { PromQueryModeller } from '@grafana/prometheus/src/querybuilder/PromQueryModeller';
```

Comment:
yet another thing that we'll need to export from `@grafana/prometheus` when Explore Metrics is externalized